Gcp-pca-section1-design-planning.md

```
@@ -0,0 +1,824 @@
+# Google Cloud Professional Cloud Architect 試験対策ガイド
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)